wiki/Guides/osu!mania_modding_guide/en.md

```diff
@@ -111,7 +111,7 @@ On the **Difficulty** page, check if the OD/HP for the beatmap follow the **offi
 
 Besides that, make sure the HP/OD used in the beatmap are appropriate for its difficulty / patterning and match up with the rest of the set. If the beatmap uses a lot of long notes for example, the OD rate should be kept rather low.
 
-The **Audio** and **Colors** pages are not that important as a modder.
+The **Audio** and **Colours** pages are not that important as a modder.
```

Comment:
depending on what's in the `Song Setup` window of the osu! editor, your change may be incorrect

Reply:
it is correct